src/main/java/eu/socialsensor/graphdatabases/HugeGraphCoreDatabase.java

```diff
@@ -216,68 +219,69 @@ private void buildGraphEnv(boolean clear) {
 
     @Override
     public void shortestPath(HugeVertex fromNode, Integer node) {
-        LOG.debug(">>>>>" + counter++ + " round,(from node: "
-                  + fromNode.id() + ", to node: " + node + ")");
+        LOG.debug(">>>> {} round, (from node: {}, to node: {})",
+                  counter++, fromNode.id(), node);
```

Comment:
log debug("...",  counter, fromNode.id(), node);
counter++;

Doing business operations inside the log does not seem to be a good style